boom! testing postgres_large_object-0.5.1 against stable for nightly
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! 1dbba82a6ec3471d6f308cc3b5b81f7f009e5ad2e6c9af0e0965b8c04c16c40d
boom! running `docker start -a 1dbba82a6ec3471d6f308cc3b5b81f7f009e5ad2e6c9af0e0965b8c04c16c40d`
kablam!    Compiling postgres_large_object v0.5.1 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.96 secs
boom! running `docker rm -f 1dbba82a6ec3471d6f308cc3b5b81f7f009e5ad2e6c9af0e0965b8c04c16c40d`
blam! 1dbba82a6ec3471d6f308cc3b5b81f7f009e5ad2e6c9af0e0965b8c04c16c40d
boom! running: cargo +stable test --frozen --no-run
boom! creating container for: cargo +stable test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen --no-run cargobomb`
blam! c9d66cebffc7056fbe61622db49a2eecd2bdc178a635116e8bb855e0cdd05c80
boom! running `docker start -a c9d66cebffc7056fbe61622db49a2eecd2bdc178a635116e8bb855e0cdd05c80`
kablam!    Compiling postgres_large_object v0.5.1 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 1.87 secs
boom! running `docker rm -f c9d66cebffc7056fbe61622db49a2eecd2bdc178a635116e8bb855e0cdd05c80`
blam! c9d66cebffc7056fbe61622db49a2eecd2bdc178a635116e8bb855e0cdd05c80
boom! running: cargo +stable test --frozen
boom! creating container for: cargo +stable test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen cargobomb`
blam! fcf3d587b4a43749956c266956ebf71e09332be924f7555414a759a705653875
boom! running `docker start -a fcf3d587b4a43749956c266956ebf71e09332be924f7555414a759a705653875`
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/postgres_large_object-4f90d697b7881e75
blam! 
blam! running 8 tests
blam! test test::test_open_bogus ... FAILED
blam! test test::test_create_delete ... FAILED
blam! test test::test_open_finish ... FAILED
blam! test test::test_delete_bogus ... FAILED
blam! test test::test_truncate ... FAILED
blam! test test::test_seek_tell ... FAILED
blam! test test::test_write_with_read_fd ... FAILED
blam! test test::test_write_read ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- test::test_open_bogus stdout ----
blam! 	thread 'test::test_open_bogus' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', ../src/libcore/result.rs:837
blam! 
blam! ---- test::test_create_delete stdout ----
blam! 	thread 'test::test_create_delete' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', ../src/libcore/result.rs:837
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- test::test_open_finish stdout ----
blam! 	thread 'test::test_open_finish' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', ../src/libcore/result.rs:837
blam! 
blam! ---- test::test_delete_bogus stdout ----
blam! 	thread 'test::test_delete_bogus' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', ../src/libcore/result.rs:837
blam! 
blam! ---- test::test_truncate stdout ----
blam! 	thread 'test::test_truncate' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', ../src/libcore/result.rs:837
blam! 
blam! ---- test::test_seek_tell stdout ----
blam! 	thread 'test::test_seek_tell' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', ../src/libcore/result.rs:837
blam! 
kablam! error: test failed
blam! ---- test::test_write_with_read_fd stdout ----
blam! 	thread 'test::test_write_with_read_fd' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', ../src/libcore/result.rs:837
blam! 
blam! ---- test::test_write_read stdout ----
blam! 	thread 'test::test_write_read' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', ../src/libcore/result.rs:837
blam! 
blam! 
blam! failures:
blam!     test::test_create_delete
blam!     test::test_delete_bogus
blam!     test::test_open_bogus
blam!     test::test_open_finish
blam!     test::test_seek_tell
blam!     test::test_truncate
blam!     test::test_write_read
blam!     test::test_write_with_read_fd
blam! 
blam! test result: FAILED. 0 passed; 8 failed; 0 ignored; 0 measured
blam! 
boom! running `docker rm -f fcf3d587b4a43749956c266956ebf71e09332be924f7555414a759a705653875`
blam! fcf3d587b4a43749956c266956ebf71e09332be924f7555414a759a705653875
